core/domain/html_cleaner.py

```diff
@@ -25,6 +25,16 @@
 
 
 def filter_a(name, value):
+    """Returns whether the given attribute of an anchor ('a') tag should be 
+    whitelisted.
+
+    Args:
+        name: The name of the attribute.
```

Comment:
Ah, sorry @souravsingh, one more thing I missed. Can you make this:

```
        name: str. The name of the attribute.
        value: str. The value of the attribute.
```

Thanks!